build/Dockerfile.nginx

```diff
@@ -1,18 +1,40 @@
 # syntax=docker/dockerfile:1.12
+# FROM scratch AS nginx-files
```

Comment:
Can we remove the commented out lines?

Reply:
I can, I just wanted to include them for reference so we know what needs to be here once agent publishes their release.

Reply:
Gotcha. Maybe add a TODO comment to the dockerfile so that's clear?